templates/web-application/swc.build.js

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+
+// Can play with most of the important configuration option with the online playground: https://swc.rs/playground
+
+/** @type {import("@swc/core").Config} */
+export default {
+    jsc: {
+        parser: {
+            syntax: "typescript",
+            tsx: true
+        },
+        // The output environment that the code will be compiled for.
+        target: "es2022",
+        transform: {
+            react: {
+                // Use "react/jsx-runtime".
+                runtime: "automatic",
+                // Use the native "Object.assign()" instead of "_extends".
+                useBuiltins: true
+            }
+        },
+        // Import shims from an external module rather than inlining them in bundle files to greatly reduce the bundles size.
+        // Requires to add "@swc/helpers" as a project dependency
+        externalHelpers: true,
+        // Informs the compiler where to find modules.
+        baseUrl: ".",
+        // Same as TS paths to register aliases.
+        paths: {
+            "@root/*": ["src/*"]
+        }
+    },
+    module: {
+        // The output module resolution system that the code will be compiled for.
+        type: "es6",
+        // Prevent SWC from exporting the `__esModule` property.
+        strict: true,
+        // Preserve dynamic imports.
+        ignoreDynamic: true
```

Comment:
I am not sure about this config. It's kinda weird as it's part of the TS type but not the JSON schema: https://json.schemastore.org/swcrc.

Reply:
I open this issue about that concern: https://github.com/workleap/wl-foundry-cli/issues/30